tools/tf2ngraph.py

```diff
@@ -343,7 +421,16 @@ def main():
     args = prepare_argparser(allowed_formats)
     inp_format, inp_loc = filter_dict("input", args.__dict__)
     out_format, out_loc = filter_dict("output", args.__dict__)
-    output_nodes = args.output_nodes.split(',')
+    if args.output_nodes is None:
+        possible_out_nodes = infer_output_nodes(inp_format, inp_loc)
+        print(
+            "Analysed graph for possible list of output nodes. Please supply one or more output node in --output_nodes"
```

Comment:
`Analyzed`. Also break the sentense into two so that it doesn't exceed. Apply to all other longer than 8- character sentences in this file.

Also mention if more than one output node is specified, what is the delimiter? Give an example.